Yasgui/packages/yasr/src/plugins/error/extended-error.ts

```diff
@@ -0,0 +1,40 @@
+import Error from "./index";
+import Parser from "../../parsers";
+
+export default class ExtendedError extends Error {
+  async draw() {
+    const error = this.yasr.results?.getError();
+    if (error) {
+      this.yasr.resultsEl.appendChild(this.createErrorElement(error));
+    }
+  }
+
+  private createErrorElement(error: Parser.ErrorSummary): HTMLDivElement {
+    const errorResponseElement = document.createElement("div");
+    errorResponseElement.className = "error-response-plugin";
+    errorResponseElement.innerHTML = this.getErrorMessage(error);
+    return errorResponseElement;
+  }
+
+  private getErrorMessage(error: Parser.ErrorSummary): string {
+    return (
+      `<div class="error-response-plugin-header">` +
+      `<div class="error-response-plugin-error-status">${error.status ? error.status + ":" : ""} ${
+        error.statusText
+      }</div>` +
```

Comment:
When you use a string literal, you don't need to do a string concatenation with `+`